### ISSUE TITLE
Tabliss does not set the homepage for new windows on Firefox

### ISSUE THREAD

**Issue by cbarrick** [joelshepherd/tabliss#182]
On Firefox, Tabliss only overrides the "new tab page" when opening a new tab, but not the "homepage" when opening a new window.

Currently, Tabliss sets `chrome_url_overrides.newtab` to control the new tab page. AFAIU, extensions should also set `chrome_settings_overrides.homepage` to control the homepage.

I think this will work to set both the new tab page and the homepage, but I haven't tried it:

```json
"chrome_url_overrides": {
  "newtab": "index.html"
},
"chrome_settings_overrides" : {
  "homepage": "index.html"
}
```

Docs:
- [chrome_url_overrides](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/chrome_url_overrides)
- [chrome_settings_overrides](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/chrome_settings_overrides)

**Comment by cbarrick** [comment]
Thanks for the work around!

> people were complaining about tabliss controlling their new tab page.

That's an odd complaint for a new tab page extension... Should we reconsider?

It seems like very odd behavior this way. The workaround isn't very discoverable.

What was the use case?

**Comment by trickypr** [comment]
There is more information available on issue #116. We should make that button easier to find...

**Comment by joelshepherd** [comment]
Firefox now let's you choose whether Tabliss overrides your homepage or not! :tada: So I am adding this back as an option (finally)!

![image](https://user-images.githubusercontent.com/4484306/122396284-2fa35780-cfbb-11eb-9a7e-cdd23b9d72aa.png)
